src/components/home/slider/sliderModal.tsx

```diff
@@ -2,80 +2,64 @@ import * as React from 'react'
 import {FC} from 'react'
 import * as SliderParent from './style'
 import {Icon} from 'react-icons-kit'
-import {ModalProvider} from '../../App'
 import {star} from 'react-icons-kit/fa/'
 import {search} from 'react-icons-kit/feather/search'
 import {globe, video} from 'react-icons-kit/feather'
-
-export interface Movie {
-  id: number
-  title: string
-  poster_path: string
-  release_date: number
-  genres: []
-  overview: string
-  vote_average: number
-  imdb: string
-}
+import {useModal} from 'src/context/modal-context'
```

Comment:
```suggestion
import {useModal} from '~context/modal-context'
```

I recommend configuring Typescript to allow importing things this way. I can try helping with that if you are up for it. The reason being is that it's a good indicator that the file does not belong to `node_modules`.

Reply:
I already configured my typescript to use the alias `@` character but I can change it to `~` would be the same.